tests/Spice86.Tests/MachineTest.cs

```diff
@@ -224,7 +224,8 @@ public void TestSelfModifyValue(bool enableCfgCpu)
         expected[0x03] = 0xff;
         Machine machine = TestOneBin("selfmodifyvalue", expected, enableCfgCpu);
         if (enableCfgCpu) {
-            CurrentInstructions currentInstructions = machine.CfgCpu.CfgNodeFeeder.InstructionsFeeder.CurrentInstructions;
+            CurrentInstructions currentInstructions = ((Core.Emulator.CPU.CfgCpu.CfgCpu)machine.Cpu).CfgNodeFeeder
```

Comment:
Can this cast be avoided, like before ?

Reply:
See https://github.com/OpenRakis/Spice86/pull/1235#discussion_r2250195681